src/Runtime/ExecutionSession.hpp

```diff
@@ -47,6 +49,7 @@ class ExecutionSession {
   llvm::sys::DynamicLibrary _sharedLibraryHandle;
 
   // Entry point function.
+  static const std::string _entryPointNameFuncName;
```

Comment:
[Suggestion]:  _entryPointNameFuncName -> _entryPointFuncName

Reply:
Yeah, it looks a bit confusing but it is actually the name of the function that returns entry point name. Let me keep this name since `entryPointFuncName` may be confusing with the name of the entry point function.